### DISCUSSION TITLE
Filter fields in alfabetic order

### DISCUSSION THREAD

**Discussion by zvfcs** [ldproxy/ldproxy#795] [Configuration] [answered]
I'm trying to get the fields in the drop-down selector to be in alphabetic order, but I don't understand where to do it, if it's in the filterEditor.mustache or somewhere else. Can anybody point me in the right direction? 

I also would suggest to make alphabetic order the default way to list the fields.

**Comment by azahnen** [comment, marked as answer]
This looks like a regression, I think the order was alphabetic in the past. 
I did not find the time to address this in [v3.3.0](https://github.com/interactive-instruments/ldproxy/releases/tag/v3.3.0), I created #848 which should be addressed in the next bugfix release.

**Comment by zvfcs** [comment]
I'm having some problems getting the filter fields in alphabetic order again. That is, I'm having problems when I'm using include all like this:
```
- buildingBlock: QUERYABLES
  enabled: true
  included:
    - "*"
```

If I specify what fields I want to include, e.g.:
```
  included:
    - vernetype
    - navn
```
the fields are rendered alphabetic.

I want to be able to use include * as a default, so I don't have to specify the fields for every service.
